### PR TITLE
Fixed #35347 -- Clarified choice_set attribute in tutorial 2.

### PR DESCRIPTION
**Pull request by AntWarden81**
Clarify how choice_set is declared/created as a part of the tutorial

# Trac ticket number

[ticket-35347](https://code.djangoproject.com/ticket/35347#ticket)

# Branch description
Clarifying how `choice_set` function is created and where it's from

# Checklist
- [x] This PR targets the `main` branch. 
- [x] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [x] I have checked the "Has patch" **ticket flag** in the Trac system.
- [x] I have added or updated relevant **tests**.
- [x] I have added or updated relevant **docs**, including release notes if applicable.
- [x] For UI changes, I have attached **screenshots** in both light and dark modes.

### CODE REVIEW

File: docs/intro/tutorial02.txt [hunk 1]
```diff
@@ -520,6 +520,8 @@ Save these changes and start a new Python interactive shell by running
     >>> q = Question.objects.get(pk=1)
 
     # Display any choices from the related object set -- none so far.
+    # As part of Django's support for model relations, q.choice_set gets
+    # created as part of Question being declared as a foreign key for Qhoice.
```
Comment (adamzap) [inline docs/intro/tutorial02.txt:524]: "Qhoice" look like a typo. Did you mean "Choice"?

Comment (AntWarden81) [reply docs/intro/tutorial02.txt]: Whoops, bad copy paste- fixed.

**Comment by nessita** [comment]
Closing per ticket.

**Comment by nessita** [comment]
Re-opening per ticket.

**Comment by smithdc1** [comment]
I'm not sure about this. I think this is too detailed and not directly related to the learning outcomes trying to be achieved in the tutorial.

**Review by nessita — APPROVED** [review]
Thank you @AntWarden81, this looks great!

Merging soon.

**Pull request merged**